src/routes/index.js

```diff
@@ -46,6 +46,10 @@ export default function BaseRouter() {
           RouteConfig.common.map((route, index) => {
             return <Route key={index} component={() => route.component()} exact path={route.path} />;
           })}
+        {isLoggedIn &&
```

Comment:
Why we are checking this isLoggedInFlag here this means that if some wants to access this component he will get the error PageNotFound but it should be unauthorized ..